src/lib/viewers/BaseViewer.js

```diff
@@ -481,6 +482,9 @@ class BaseViewer extends EventEmitter {
      * @return {string} content url
      */
     createContentUrlWithAuthParams(template, asset) {
+        if (!template) {
```

Comment:
Might make sense to add a comment here to explain why we return empty string for this case